c2corg_ui/templates/outing/view.html

```diff
@@ -50,6 +50,9 @@
   });
 
   module.value('documentData', {
+    "document_id": ${outing.get('document_id')},
+    "lang": ${dumps(lang) | n},
```

Comment:
I agree that this is not really necessary for "lang". IMHO, it is a good practice to always use the good escaping method, if dumps has nothing to do, it will do nothing. HTML templating always do html entities escaping, even if it is not visible in the template, this is the default. I don't see any filter for json in mako documentation, but it should be possible to create one, example : ``${lang | json}``.